edu-services/gradebook-service/impl/src/java/org/sakaiproject/component/gradebook/BaseHibernateManager.java

```diff
@@ -394,7 +394,7 @@ public boolean isExplicitlyEnteredCourseGradeRecords(final Long gradebookId) {
         HibernateCallback<Boolean> hc = session -> {
             Long result = (Long) session.createCriteria(CourseGradeRecord.class)
                     .add(Restrictions.isNotNull("enteredGrade"))
-                    .add(Restrictions.eq("gradableObject.gradebook.id", gradebookId))
+                    .add(Restrictions.eq("id", gradebookId))
```

Comment:
Can see the relevant stack trace here: https://gist.github.com/ottenhoff/bf3feca3e3ddd97d06a44c0b58d647c8